journal_brief/journal_brief.py

```diff
@@ -146,17 +146,31 @@ def __init__(self, cursor_file=None, reader=None, dry_run=False,
                 self.cursor = fp.read()
         except IOError as ex:
             if ex.errno == errno.ENOENT:
-                self.cursor = None
+                self.cursor = ''
             else:
                 raise
 
         if reader is None:
             reader = journal.Reader()
 
-        if seek_cursor and self.cursor:
-            log.debug("Seeking to %s", self.cursor)
-            reader.seek_cursor(self.cursor)
-            reader.get_next()
+        if len(self.cursor) != 0:
```

Comment:
`if self.cursor:` should be fine here I think? It's only truthy if it has length.

Reply:
Indeed, this was an attempt to understand why tests were failing :-)